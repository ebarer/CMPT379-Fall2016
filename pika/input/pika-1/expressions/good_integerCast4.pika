exec {
	const v := -3.9427.
	const vapprox := [ v | int].		# don^t^ round down. give me -3
	print v; vapprox, _n_.
}