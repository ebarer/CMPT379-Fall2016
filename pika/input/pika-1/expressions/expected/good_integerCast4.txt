-3.9427 -3
